source/draw/DrawContext.ooc

```diff
@@ -39,9 +39,10 @@ DrawContext: abstract class {
 	createYuv420Semiplanar: abstract func ~fromImages (y, uv: Image) -> Image
 	createYuv420Semiplanar: abstract func ~fromRaster (raster: RasterYuv420Semiplanar) -> Image
 	createImageFromTextAndFont: virtual func (message: Text, fontAtlas: Image) -> Image {
-		target := this createMonochrome(This getTextPixelBounds(message, fontAtlas))
+		target := this createMonochrome(This getTextPixelBounds(message take(), fontAtlas))
```

Comment:
I think typical use case is to save the result of `take()` to a variable and use that variable instead of calling `take()` multiple times. @marcusnaslund I think there is a user manual for this somewhere ?